sources/packages/web/src/components/fa-application/ConfirmStudentAidBCProfileInfo.vue

```diff
@@ -20,8 +20,8 @@
         <label for="gender" class="label-extra">Gender</label>
         <div class=" p-col text-extra">{{ studentConfirmInfo.gender }}</div>
       </div>
-    </div>
-    <h4 class="p-mb-5">If the Information is incorrect, please <a href=#>click here</a></h4>
+    </div>    
+    <h4 class="p-mb-5">If the Information is incorrect, please <router-link :to="{ name: 'Student-Profile-Edit'}">click here</router-link></h4> 
```

Comment:
It looks good 👍 
At some point in the future let's start to create constants and use these constants for router index and redirects.